pkg/components/components.go

```diff
@@ -29,7 +29,7 @@ var (
 )
 
 const (
-	MultusImageDefault            = "ghcr.io/k8snetworkplumbingwg/multus-cni@sha256:829c27e9392d013eee5086ca7670d7326d723ebaec526237215e86086b5a3234"
+	MultusImageDefault            = "ghcr.io/k8snetworkplumbingwg/multus-cni:v3.9.1-62-g1aac2431"
```

Comment:
CNAO is now working with sha digests in U/S, so we need to keep using the sha. 
Usually the last part of the bump script is handling getting the sha and replacing it here - so I would try to check why that part didn't work..

Reply:
well, if we're specific I think we need commit's image to be pushed to `ghcr.io/k8snetworkplumbingwg/multus-cni`. usually push occurs only for tagged commits via some post script, but some repos do it for every commit (kmp for [example](https://quay.io/repository/kubevirt/kubemacpool?tab=tags&tag=latest)) so that's an option as well, albeit tagged is the preferred way to go..

Reply:
Check the next commit: https://github.com/kubevirt/cluster-network-addons-operator/pull/1433/commits/c3ca44bc17488cf56f966eaf11b6ae7fa9037c9d#diff-f1531df30500812ec42a33a52023c9ceca8a93d4972132fd241ad143b107c063R32

The image exists, since, as you say, multus pushes all imgs to their container registry. What's broken is we [look for the tags attached to a commit](https://github.com/kubevirt/cluster-network-addons-operator/blob/main/hack/components/bump-multus.sh#L127); when none is found, somehow we create that weird thing.

Reply:
ah yes you're right. when the tag is not there, it will use a virtual tag thing, cause this is how we tag it on [quay](https://quay.io/repository/kubevirt/kubemacpool?tab=tags&tag=latest) as well.. well if you need to change it to fit to how multus will now be used (w/wo tags) then feel free to do so :) .